stdlib/public/core/Sort.swift.gyb

```diff
@@ -140,7 +140,7 @@ func _introSort<C>(
   var areIncreasingVar = areInIncreasingOrder
 %   end
   let count =
-    elements.distance(from: range.lowerBound, to: range.upperBound).toIntMax()
+    elements.distance(from: range.lowerBound, to: range.upperBound)
```

Comment:
Maybe just drop the named constant now that we can do the comparison without type coercion.